modules/logging-rn-5.7.3.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-3_{context}"]
+= Logging 5.7.3
+This release includes link:https://access.redhat.com/errata/RHSA-2023:3998[OpenShift Logging Bug Fix Release 5.7.3].
+
+[id="openshift-logging-5-7-3-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, when viewing logs within the {product-title} web console, cached files caused the data to not refresh. With this update the bootstrap files are not cached, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4100[LOG-4100])
+
+* Before this update, the Loki Operator reset errors in a way that made identifying configuration problems difficult to troubleshoot. With this update, errors persist until the configuration error is resolved. (link:https://issues.redhat.com/browse/LOG-4156[LOG-4156])
+
+* Before this update, the LokiStack ruler did not restart after changes were made to the `RulerConfig` CR. With this update, the Loki Operator restarts the ruler pods after the `RulerConfig` CR is updated. (link:https://issues.redhat.com/browse/LOG-4161[LOG-4161])
+
+* Before this update, the vector collector terminated unexpectedly when input match label values contained a `/` character within the `ClusterLogForwarder`. This update resolves the issue by quoting the match label, enabling the collector to start and collect logs. (link:https://issues.redhat.com/browse/LOG-4176[LOG-4176])
+
+* Before this update, the Loki Operator terminated unexpectedly when a `LokiStack` CR defined tenant limits, but not global limits. With this update, the Loki Operator can process `LokiStack` CRs without global limits, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4198[LOG-4198])
+
+* Before this update, Fluentd did not send logs to an Elasticsearch cluster when a passphrase applied to the private key provided. With this update, Fluentd properly handles passphrase-protected private keys when establishing a connection with Elasticsearch. (link:https://issues.redhat.com/browse/LOG-4258[LOG-4258])
```

Comment:
- First sentence seems to be missing a word or two because it doesn't make sense to me as written - `when a passphrase applied to the private key provided`. Should it be `when a passphrase applied to the private key was provided`?